smart_open/webhdfs.py

```diff
@@ -1,9 +1,12 @@
-#!/usr/bin/env python
+#
 # -*- coding: utf-8 -*-
 #
+# Author: Michael Penkov <m@penkov.dev>
```

Comment:
The `Author:` doesn't seem always accurate, so let's drop it altogether. Like you say, `git blame` will be more up-to-date.